app/services/ReviewService.scala

```diff
@@ -64,31 +64,61 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
   def changeStatus(id: String, currentStatus: String, info: ApprovalProcessStatusChange): Future[RequestOutcome[Unit]] = {
 
     def getContentToUpdate: Future[Option[ApprovalProcess]] = repository.getById(id) map {
-        case Right(process) => if (process.meta.status == currentStatus) Some(process) else None
-        case _ => None
-      }
+      case Right(process) =>
+        if (process.meta.status == currentStatus) {
```

Comment:
clearer as 
case Right(process) if process.meta.status == currentStatus =>
case Right(process) =>
case Left(err) =>

Reply:
thanks, it didn't seem the right way to structure it

Reply:
Done, in next commit